pdaltools/add_points_in_pointcloud.py

```diff
@@ -147,13 +260,46 @@ def add_points_from_geojson_to_las(
         if spatial_ref is None:
             raise RuntimeError(f"LAS file {input_las} does not have a valid EPSG code.")
 
+    # Read the input GeoJSON
+    gdf = gpd.read_file(input_geometry)
+
+    if gdf.crs is None:
+        gdf.set_crs(epsg=spatial_ref, inplace=True)
+
+    # Clip lines to the LIDAR tile
+    gdf = clip_3d_lines_to_tile(gdf, input_las, spatial_ref, tile_width)
+
+    # Check if both Z in geometries and altitude_column are provided
+    if gdf.geometry.has_z.any() and altitude_column:
+        raise ValueError("Both Z in geometries and altitude_column are provided. Please provide only one.")
+
+    # Store the unique geometry type in a variable
+    unique_geom_type = gdf.geometry.geom_type.unique()
+
+    # Check the geometry type
+    if len(unique_geom_type) != 1:
+        raise ValueError("Several geometry types found in geometry file. This case is not handled.")
+
+    if unique_geom_type == ["Point"] or unique_geom_type == ["MultiPoint"]:
```

Comment:
detail : ça peut aussi marcher comme ça, qui me parait plus clair : 
```suggestion
    if unique_geom_type in ["Point", "MultiPoint"]:
```